plugin/trino-hive-hadoop2/pom.xml

```diff
@@ -96,6 +96,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>io.trino.tpch</groupId>
+            <artifactId>tpch</artifactId>
+            <scope>test</scope>
```

Comment:
I think we should merge in the other direction. `trino-hive` should contain all the tests, not `trino-hive-hadoop2`.

Reply:
Ok I had that doubt, but thought i will start with minimal change.
Will move it.

Reply:
This would require moving HivePlugin to trino-hive. Is that ok ? Or should i refactor test to use TestHivePlugin ?
I would prefere move as it looks it will not bring much impact, but not sure.

Reply:
For now tried to refactor to use TestHivePlugin